devel/pulp/devel/test_runner.py

```diff
@@ -60,9 +69,28 @@ def run_tests(packages, tests_all_platforms, tests_non_rhel5):
                      ','.join(packages)])
 
     # don't run the server or plugins tests in RHEL5.
+    flake8_exit_code = 0
     if sys.version_info >= (2, 6):
+        # make sure we test everything
         args.extend(tests_non_rhel5)
+
+        # Check the files for coding conventions
+        if run_flake8:
+            if not flake8_paths:
+                flake8_paths = ['.']
```

Comment:
The other option is to require these paths to be absolute paths, and to require them to be present. Thus, it is the caller's responsibility to make sure the paths do not depend on CWD.